Cache/Hashing.lean

```diff
@@ -118,7 +128,13 @@ partial def getHash (filePath : FilePath) (visited : Std.HashSet FilePath := ∅
       modify fun stt => { stt with cache := stt.cache.insert filePath none }
       return none
     let content ← IO.FS.readFile fixedPath
-    let fileImports := getFileImports content (← getPackageDirs)
```

Comment:
If `getPackageDirs` is no longer used anywhere, I think you should drop it in this PR at the same time.

Reply:
It is still used in `getPackageDir` ([here](https://github.com/leanprover-community/mathlib4/blob/3c3b1965ec6415f3b5483f4f89c2dcd6a1fc4c4e/Cache/IO.lean#L156)) which in turn is used [here](https://github.com/leanprover-community/mathlib4/blob/3c3b1965ec6415f3b5483f4f89c2dcd6a1fc4c4e/Cache/Hashing.lean#L121).

Reply:
Since there are two PRs on the queue both removing one of the two usages (the other being #21817), there is the follow-up PR #21818 which removes `getPackageDirs`, .

Reply:
Sounds good to me

Reply:
done